components/director/internal/domain/subscription/service.go

```diff
@@ -28,6 +29,13 @@ type Config struct {
 	RuntimeTypeLabelKey        string `envconfig:"APP_RUNTIME_TYPE_LABEL_KEY,default=runtimeType"`
 }
 
+const (
+	// SubdomainLabelKey is the key of the tenant label for subdomain.
+	SubdomainLabelKey = "subdomain"
+	// RegionPrefix a prefix to be trimmed when creating an app from template
```

Comment:
RegionPrefix a prefix to be trimmed **from the region placeholder value** when creating an app from template